3rdparty/ErrorHandling/errorcodes.h

```diff
@@ -3,31 +3,26 @@
 
 #include <map>
 
-struct ErrorCodes
-{
-    enum Errors {
-        UnsupportedFormat,
-        FileNotFound,
-        Blank,
-        ParsemzXml,
-        ParsemzMl,
-        ParsemzData,
-        ParseCsv
-    };
-
-};
-
 //singleton
 class ErrorMsg
 {
     public:
+        enum Errors {
```

Comment:
@rish9511 Why are we not assigning any integer value?

Reply:
They all assigned an Integer value implicitly. 
You want to assign some other values to them ?

Reply:
@rish9511 Just for convenience for developers. This is also fine.